data/mods/No_Hope/Mapgen/mil_base_z0.json

```diff
@@ -844,8 +845,8 @@
         { "vehicle": "military_vehicles", "x": 9, "y": 32, "chance": 25, "status": 1 },
         { "vehicle": "military_vehicles", "x": 20, "y": 39, "chance": 25, "status": 1 },
         { "vehicle": "military_vehicles", "x": 83, "y": 39, "chance": 25, "rotation": 90 },
-        { "vehicle": "welding_cart", "x": 67, "y": 36, "chance": 80 },
-        { "vehicle": "engine_crane", "x": 55, "y": 26, "chance": 80 }
+        { "vehicle": "welding_cart", "x": 67, "y": 36, "chance": 80, "status": 0 },
+        { "vehicle": "engine_crane", "x": 55, "y": 26, "chance": 80, "status": 0 }
```

Comment:
> Comment from Night-Pryanik:
>"The same here." (referring to: "Not needed. The default status is -1, which means it's lightly damaged. I see no reason for a forklift to be ideally undamaged with status 0.")

@Night-Pryanik 

Same as for the forklift. Found in buildings closed & locked since the cataclysm with other undamaged equipment. Why would these be the only equipment damaged in them?
Again, if you disagree, I'll set status to -1.